submissions/omolayoOdupe/todo_app/src/components/InputTodo.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const InputTodo = (props) => {
+
+    return(
+        <form className='to-do-form'>
+            <input
+                value ={props.value}
+                type={props.inputType}
```

Comment:
I think you should add this type by default. There is no need to have it passed in as props, same for placeholder @OmolayoO